### PR TITLE
updated seed file to match new DB schema

### PR DESCRIPTION
**Pull request by bbuilds**
updated seed file

### CODE REVIEW

File: seed/index.js [hunk 1]
```diff
@@ -1,12 +1,24 @@
 require("dotenv").config()
 
 const SpoofItem = require("./../api/models/spoofitem")
-const mongoose = require("mongoose")
-mongoose.connect(process.env.MONGODB_URI, {
-  useMongoClient: true
+const {mongoose} = require('./../config/mongoose')
+const Counter = require("./../api/models/counter")
+
+
```
Comment (kyleva) [inline seed/index.js:7]: I don't think `SpoofItem.remove` is synchronous. The create queries below could run before this clears the DB.

File: seed/index.js [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
+//Clear Collections to start new + add counter
+SpoofItem.remove({}).then((result) => {
+  console.log('spoof items cleared')
+})
+
```
Comment (kyleva) [inline seed/index.js]: I don't think `Counter.remove` is synchronous. The create query can execute before the DB has been cleared.

File: seed/index.js [hunk 3]
```diff
@@ -1,12 +1,24 @@
...
+Counter.remove({}).then((result) => {
+  console.log('counters cleared')
 })
-const db = mongoose.connection
-db.on("error", console.error.bind(console, "MongoDB connection error:"))
+
```
Comment (kyleva) [inline seed/index.js]: Consider using `Promise.all` to wait until all promises have resolved and then populate the database. Here's an example: https://gist.github.com/kyleva/00d5a12414afe0ec46e513b3241e0471

File: seed/index.js (hunk 1 above)
Comment (bbuilds) [reply seed/index.js:7]: updated with promise.all

**Review by kyleva — APPROVED** [review]

**Pull request merged**